packages/fx-core/src/plugins/resource/aad/aadAppClient.ts

```diff
@@ -47,7 +47,10 @@ export class AadAppClient {
       config.clientId = provisionAadResponse.appId;
       config.objectId = provisionAadResponse.id;
     } catch (error) {
-      throw ResultFactory.SystemError(CreateAppError.name, CreateAppError.message(), error);
+      if (error?.response?.data?.errorMessage.includes(Constants.appStudioErrorMessage)) {
```

Comment:
can we treat 5xx error as system error?

Reply:
Yeah I think we can classify error type by status code in error message. I will update logic here.